pkg/cliplugins/bind/cmd/cmd.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	"github.com/kcp-dev/kcp/pkg/cliplugins/bind/plugin"
+)
+
+var (
+	// TODO: add other examples related to permission claim commands.
+	bindExampleUses = `
+	# binds to the mentioned workspace in the command, e.g the below command will create
+	# an APIBinding named "my-binding", that binds to "my-export" present in "root:my-export-service" workspace.
```

Comment:
Since this is an example, I would suggest having the comment strictly limited to describing what the command is doing:

```shell
# Create an APIBinding named "my-binding" that binds to the APIExport "my-export" in the "root:my-service" workspace.
```